java/ql/lib/semmle/code/java/security/PathSanitizer.qll

```diff
@@ -357,6 +357,21 @@ private predicate maybeNull(Expr expr) {
   )
 }
 
+/** A taint-tracking configuration for reasoning about tainted arguments. */
+private module TaintedArgConfig implements DataFlow::ConfigSig {
+  predicate isSource(DataFlow::Node src) {
+    src instanceof ActiveThreatModelSource or
+    src instanceof ApiSourceNode or
+    // for InlineFlowTest
+    src.asExpr().(MethodCall).getMethod().getName() = "source"
```

Comment:
This is fine for testing, but we shouldn't commit this. I'm sure we can find some other solution, like adding a model to the inlineflowtest so that `source` is a remote flow source.

Reply:
Can you clarify what you mean by "adding a model to the inlineflowtest"? I was thinking of using an actual `ActiveThreatModelSource` in this test case instead of `(File) source();`. Is that what you mean? Or did you have something else in mind?

Reply:
I meant you could add a model for `source` as a remote flow source that applies just to this test (if the test is `x.ql` then the model has to be in a file called `x.ext.yml` in the same folder). That way the inline flow test can use its way of finding sources (calls to "source") and this flow config can use its way (active threat model). An alternative approach is to change the test to use `ActiveThreatModel` for its sources, which I think is what you were suggesting. Both are fine.